### PR TITLE
Icu custom data

### PR DESCRIPTION
**Pull request by mgautierfr**

### CODE REVIEW

**Comment by mgautierfr** [comment]
Please @kelson42 review this PR. Especially this commit https://github.com/kiwix/kiwix-build/pull/44/commits/1109baec07f942548db576f684748aa961c78a5f

It's was you who has generated the previous `icudata56l.dat` so you have a clue on what to keep and what can be dropped.

File: dependencies.py [hunk 1]
```diff
@@ -184,7 +184,8 @@ class Source(SvnClone):
         svn_dir = "icu4c"
 
         patches = ["icu4c_fix_static_lib_name_mingw.patch",
-                   "icu4c_android_elf64_st_info.patch"]
+                   "icu4c_android_elf64_st_info.patch",
+                   "icu4c_custom_data.patch"]
```
Comment (kelson42) [inline dependencies.py]: What is the problem here, why do we need to patch ICU?

**Comment by kelson42** [comment]
@mgautierfr In general looks good.

**Comment by kelson42** [comment]
The only thing I can say about it comes from this comment:
https://github.com/kiwix/kiwix/blob/master/src/dependencies/Makefile.am#L247

```
# This files gathers ICU related data. It's a custom one generated
# with http://apps.icu-project.org/datacustom/ICUData56.html. It
# contains Break iterators, Collators and Transliterators.
```

**Pull request merged**